README.md

```diff
@@ -16,13 +16,13 @@ Tutorial
 
 Usage
 ---
-In this example, we read the realtime Bitcoin price from [Gdax WebSocket Feed][gdax-websocket-feed].
+In this example, we read the realtime Bitcoin price from [Coinbase WebSocket Feed][https://docs.pro.coinbase.com/].
 For more information, please check out the [demo app][demo-app].
```

Comment:
we'll probably need to update the demo to make it consistent

Reply:
@aaronhe42 I think so. Do you want me to update the demo accordingly?